guides-for-using-cinchy/builder-guides/creating-tables/columns.md

```diff
@@ -194,47 +197,64 @@ Cinchy has several Date column type display format options available:
+{% endhint %}
+### Live vs Cached Calculated Columns
 
-When creating a calculated column, you will notice the option to have it cached or not, located under Advanced Settings. This was an option introduced in version 4.0 of the platform.
+#### Choose Your Calculation Type
 
-![](<../../../.gitbook/assets/image (304).png>)
+When creating a calculated column, you have two types to choose from: **cached** and **live**. This feature is accessible via the Advanced Settings and was a part of the 4.0 version update.
 
-A **cached calculated column** stores your data for fast retrieval and querying. Calculated columns are defaulted to cached. This is an actual column in your table that's based on a defined CQL formula. This column will recalculate when data is changed **in the same row.**
+#### Cached Calculated Columns
 
-In the below example, the **Label** column is a calculated column that connects various name columns together. You can see that _Connect your data_ appears in each label. If you then wanted to change the name in row one from _Connect your data_ to _Connect **all** your data_, only that specific row would recalculate automatically to update the label. To update any other row (within this table or another) with a calculated column that references that data, you would need to manually make changes to each to prompt a recalculation.
+- **What It Does**: Speeds up data retrieval.
+- **How It's Stored**: As an actual column based on a CQL formula.
+- **When It Updates**: Updates only if the data in the same row changes.
 
-![](<../../../.gitbook/assets/image (205).png>)
+**Example**:  
 
-The other option is to deselect the cached button, and to have a **live calculated column** instead. A live column runs the query in real time, on the fly: it's a stored formula that's only executed when you read or query the record, and is executed each time you do so.
+Changing a name in a single row only triggers a recalculation for that row's "Label" column.
 
-In the above example, if the **Label** column was a live calculation, the formula would run every time you refresh the screen or query the data. If changes have been made (in this example, changing _"Connect your data"_ to _"Connect **all** your data"_), the calculated column will automatically reflect them, regardless of if it references another row or even another table.
+#### Live Calculated Columns
 
-While live calculated columns can be powerful this way, they also have their drawbacks, including possibly throwing errors and being resource intensive. Sometimes live calculated columns can even break a table (for example if you have a live calculated column linking to **another** live calculated column, you may run into errors).
+- **What It Does**: Provides real-time data.
+- **How It's Stored**: As a formula executed on-the-fly during read or query.
+- **When It Updates**: Refreshes automatically upon every query or screen refresh.
 
-### Choice
+**Example**: 
```

Comment:
Similar to above, maybe we can provide some more opinionated guidance:

**When to use:**
-  Your calculated column depends on a value from a linked table and you always need the latest value from the linked table.
- Your table does not contain a large number of records _(how small? Maybe QA would know)_